server/server.js

```diff
@@ -54,12 +60,68 @@ app.use(passport.initialize());
 app.use(passport.session());
 
 /*** APIs ***/
+
 app.get('/api/team', (req, res) => {
     dao.getTeam()
         .then((team) => { res.json(team); })
         .catch((error) => { res.status(500).json(error); });
 });
 
+app.get('/api/services', (req, res) => {
+    dao.getAllServices()
+        .then((services) => { res.json(services); })
+        .catch((error) => { res.status(500).json(error); });
+});
+
+app.get('/api/services/:id', (req, res) => {
+    const id = req.params.id;
+    dao.getService(id)
+        .then((service) => { res.json(service); })
+        .catch((error) => { res.status(500).json(error); });
+});
+
+app.post('/api/services', /* isAdmin, */ async (req, res) => {
+    const name = req.body.name;
+    const time = req.body.time;
+
+    try {
+        await dao.createService({
+            name: name,
+            time: time
+        });
+        res.end();
+    } catch (error) {
+        res.status(500).json(error);
+    }
+});
+
+app.put('/api/services/:id', /* isAdmin, */ async (req, res) => {
+    const id = req.params.id;
+    const name = req.body.name;
+    const time = req.body.time;
+
+    try {
+        await dao.modifyService({
+            id: id,
+            name: name,
+            time: time
+        });
+        res.end();
+    } catch (error) {
+        res.status(500).json(error);
+    }
+});
+
+app.delete('/api/services/:id', /* isAdmin, */ async (req, res) => {
+    const id = req.params.id;
+    try {
+        await dao.deleteService(id);
+        res.end();
+    } catch (error) {
+        res.status(500).json(error);
+    }
+});
+
```

Comment:
Not useful for the story, i suggest to remove it before merging the code.

Reply:
removed.